app/models/loan_filtered_question.rb

```diff
@@ -31,7 +35,7 @@ def required?
     if override_associations || depth == 1
       loan_types.include?(loan.loan_type_option)
     else
-      parent&.required?
+      parent.required?
```

Comment:
Why remove this?

Reply:
oops...was debugging failing tests and forgot to undo that change

out of curiosity though, what does that do?

Reply:
This ended up having an impact on @arisah-ijeoma's code. See 
https://github.com/sassafrastech/madeline/pull/380#pullrequestreview-62372571. So this code change actually needs to be discussed further.